docs/Changelog.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+1.15.0 (Not released yet)
+* [#5724](https://github.com/TouK/nussknacker/pull/5724) Improvements: Allow Designer to run locally with remote debugging & normalize SCALA_VERSION env param name
```

Comment:
I think that we shouldn't expose this in changelog. It is interesting for developers instead of users. Instead, we should mention about that in contributing guide